CHANGELOG.md

```diff
@@ -25,21 +25,46 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Security
 -->
 
-## [Unreleased](https://github.com/CDRH/datura/compare/v0.2.0-beta...dev)
+## [1.0.0](https://github.com/CDRH/datura/compare/v0.2.0-beta...dev)
 
 ### Added
 - minor test for Datura::Helpers.date_standardize
 - documentation for web scraping
 - documentation for CsvToEs (transforming CSV files and posting to elasticsearch)
+- documentation for adding new ingest formats to Datura
+- byebug gem for debugging
 - instructions for installing Javascript Runtime files for Saxon
+- API schema can either be 1.0 or 2.0 (which includes nested fields); 1.0 will be run by default unless 2.0 is specified. Add the following to `public.yml` or `private.yml` in the data repo:
+```
+api_version: '2.0'
+```
+- schema validation with API version 2.0, invalidly constructed documents will not post
+- authentication with Elasticesarch 8.5; add the following to `public.yml` or `private.yml` in the data repo:
+```
+  es_user: username
+  es_password: ********
+```
```

Comment:
Instructions like these to add / edit config files should be in the Migration section. Otherwise the Migration section looks very thorough :+1: I'll have to edit this file for merge conflicts after I updated the changelog file to fix up the last release though. I'll take care of it during review of the `release/v1.0.0` branch which will have the merge conflict.